spec/support/rails_template.rb

```diff
@@ -48,9 +48,14 @@ def display_name
     "#{first_name} #{last_name}"
   end
 }, after: 'class User < ActiveRecord::Base'
+copy_file File.expand_path('../templates/user_decorator.rb', __FILE__), "app/models/user_decorator.rb"
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.